### ISSUE TITLE
from rdkit import Chem error

### ISSUE THREAD

**Issue by swapchavan** [rdkit/rdkit#1236]
Hi, 
I am using conda on windows (64bit) and having hand-on for RDKit tutorial. I am using python 3.5.2. I have installed postgresql, Microsoft visual c++ (2015), numpy, matplotlib, pillow, win32all and libraries boost, cmake and rdkit-Release_2016_09_3.
I am trying to import Chem from rdkit, and I am not able to import it (see attached image for details).
I didn't got any error for the following
from rdkit import DataStructs
Kindly help me

Thanks in advance
Swapnil
![rdkit_chem_error](https://cloud.githubusercontent.com/assets/24858456/21579825/b62a9d00-cf92-11e6-9f44-e3c54c635680.PNG)

**Comment by tentrillion** [comment]
I brought this up on the rdkit-discuss@lists.sourceforge.net email list.  @greglandrum supplied a quick fix:

<blockquote> If you change lines 32 and 33 in <Anaconda>/lib/site-packages\rdkit\RDConfig.py
to:
  condaDir += ['Library', 'share', 'RDKit']
  _share = os.path.join(*condaDir)

I think it should work.

Sorry for the inconvenience here; we will fix it before running the next conda builds.

-greg
</blockquote>

The root cause seems to be that the anaconda Windows build is out of sync with the main codebase.  As Matt Swain explained on the same email list:

<blockquote>
I've also encountered this problem with the 2016.09.2 windows packages on the rdkit conda channel. It looks like somehow the RDConfig patch in the conda recipe hasn't been applied properly in the published packages.

The original lines in the rdkit are:

    condaDir += ['share', 'RDKit']
    _share = os.path.join(*condaDir)

The conda recipe has a Windows-specific patch to change this to:

    condaDir += ['Library','share','RDKit']
    _share = os.path.sep.join(condaDir)

Which looks fine (although the second line doesn't really need changing?). But in the published packages it is:

    condaDir += ['share', 'RDKit', 'RDKit']
    _share = os.path.join(condaDir)

This causes the AttributeError because it incorrectly passes a list to os.path.join, with no asterisk for unpacking the list into *args. The first line is also incorrect.

I built the package myself from the recipe, and didn't see this issue.
</blockquote>

**Comment by swapchavan** [comment]
I solved it.I search for available distributions on conda and then installed it from that particular distribution. When it asked me about dependencies I checked yes and it has installed all the required packages.Thanks for your answer though.Have a nice weekend/Swap  

    On Thursday, January 19, 2017 9:08 AM, tentrillion <notifications@github.com> wrote:
 

 I brought this up on the rdkit-discuss@lists.sourceforge.net email list. @greglandrum supplied a quick fix:
 If you change lines 32 and 33 in /lib/site-packages\rdkit\RDConfig.pyto: condaDir += ['Library', 'share', 'RDKit'] _share = os.path.join(*condaDir)I think it should work.Sorry for the inconvenience here; we will fix it before running the next conda builds.-greg
The root cause seems to be that the anaconda Windows build is out of sync with the main codebase. As Matt Swain explained on the same email list:
I've also encountered this problem with the 2016.09.2 windows packages on the rdkit conda channel. It looks like somehow the RDConfig patch in the conda recipe hasn't been applied properly in the published packages.The original lines in the rdkit are:condaDir += ['share', 'RDKit']
_share = os.path.join(*condaDir)
The conda recipe has a Windows-specific patch to change this to:condaDir += ['Library','share','RDKit']
_share = os.path.sep.join(condaDir)
Which looks fine (although the second line doesn't really need changing?). But in the published packages it is:condaDir += ['share', 'RDKit', 'RDKit']
_share = os.path.join(condaDir)
This causes the AttributeError because it incorrectly passes a list to os.path.join, with no asterisk for unpacking the list into *args. The first line is also incorrect.I built the package myself from the recipe, and didn't see this issue.
